src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -6089,12 +6089,13 @@ def test_aks_disable_addon_web_app_routing(self, resource_group, resource_group_
             # self.check('ingressProfile.webAppRouting.enabled', False)
         ])
 
+    # graph api is not well mocked
     @live_only()
     @AllowLargeResponse()
     @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
     def test_aks_create_and_update_web_application_routing_dns_zone(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=76684322&view=results) to validate the change.